deepfence_server/reporters/scan/scan_result_actions.go

```diff
@@ -61,6 +61,15 @@ func UpdateScanResultMasked(ctx context.Context, req *model.ScanResultsMaskReque
 		return err
 	}
 	defer tx.Close()
+	getImageStubQuery := func() string {
+		imageStubQuery := ""
+		if value == true {
```

Comment:
As indicated ;)

Reply:
Removed the local function and moved the code inline.